notify/api/v1/api.py

```diff
@@ -15,21 +15,44 @@
 
 import logging
 
+import jsonschema
 import flask
 
 from notify import config
-
+from notify.drivers.salesforce import salesforce
 
 LOG = logging.getLogger("api")
 LOG.setLevel(config.get_config().get("logging", {}).get("level", "INFO"))
 
+notify_backends_config = config.get_config()['notify_backends']
+sfdc_config = notify_backends_config['sf']['salesforce']['properties']
+
+
+sfdc_oauth2 = salesforce.OAuth2(
+    client_id=sfdc_config['client_id'],
+    client_secret=sfdc_config['client_secret'],
+    username=sfdc_config['username'],
+    password=sfdc_config['password'],
+    auth_url=sfdc_config['auth_url'],
+    organizationId=sfdc_config['organization_id'])
+
+sfdc_client = salesforce.Client(sfdc_oauth2)
 
 bp = flask.Blueprint("alert", __name__)
 
 
-@bp.route("/alert", methods=["POST"])
-def send_alert(period):
-    return flask.jsonify({})
+@bp.route("/notify/sfdc", methods=["POST"])
+def send_alert():
+    content = flask.request.get_json(force=True)
```

Comment:
you should first do this: 

```
notify_bends =  config.get_config()['notify_backends']
for backend in backends.split(","):
   not_supported_backends = []
   if backend in notify_bends: 
      if "salesforce" in notify_bends[backend]: 
           #do your sf code
      else: 
            not_supported_backends.append((backend, notify_backends[backend]))

     return code 409 or 200 depending on not_supporeted_backends value
```
you should iterate here over config.get_config()['notify_backends'] keys and check if keys match